stackslib/src/net/api/postblock_proposal.rs

```diff
@@ -538,6 +538,7 @@ impl NakamotoBlockProposal {
         }
 
         let mut block = builder.mine_nakamoto_block(&mut tenure_tx);
+        block.header.version = self.block.header.version;
```

Comment:
Nit: should have done this in the other PR...could you add a comment here cause I could see this getting deleted without realizing its needed.